mount.go

```diff
@@ -95,9 +96,14 @@ func mount(source, destination, fsType string, flags int, options string) error
 		return fmt.Errorf("need mount FS type")
 	}
 
-	if fsType != type9pFs {
-		var err error
-
+	var err error
```

Comment:
nit: you dont need to declare err here, since you use a new instance in the inner scope.

Reply:
yes